package.json

```diff
@@ -900,20 +900,24 @@
     "vscode:prepublish": "npm run compile",
     "compile": "tsc -p ./",
     "lint": "eslint src --ext .ts",
-    "watch": "tsc -watch -p ./"
+    "watch": "tsc -watch -p ./",
+    "pretest": "npm run compile",
+    "test": "node ./out/test/runTest.js"
   },
   "dependencies": {
-    "tmp": "0.0.31",
+    "tmp": "0.2.1",
     "vscode-nls": "^2.0.1"
   },
   "devDependencies": {
+    "@types/mocha": "^7.0.2",
     "@types/node": "^10.17.24",
-    "@types/tmp": "0.0.33",
+    "@types/tmp": "0.2.0",
     "@types/vscode": "^1.30.0",
-    "typescript": "^3.8.3",
-    "vscode-test": "^1.4.0",
-    "eslint": "^6.8.0",
+    "@typescript-eslint/eslint-plugin": "^2.30.0",
     "@typescript-eslint/parser": "^2.30.0",
-    "@typescript-eslint/eslint-plugin": "^2.30.0"
+    "eslint": "^6.8.0",
+    "mocha": "^7.2.0",
```

Comment:
Can we keep the order of the dependencies that were not modified?

Reply:
Yes, I'll fix this. I installed using `npm install --save` and it likes rearranging things, but if we want to alphabetize stuff, we should do it in a separate commit.